database/engine/rrdenginelib.c

```diff
@@ -187,10 +184,10 @@ char *get_rrdeng_statistics(struct rrdengine_instance *ctx, char *str, size_t si
               "global_flushing_pressure_page_deletions: %ld\n",
               (long)ctx->stats.metric_API_producers,
               (long)ctx->stats.metric_API_consumers,
-              (long)pg_cache->page_descriptors,
+              0L,
               (long)ctx->stats.page_cache_descriptors,
-              (long)pg_cache->populated_pages,
-              (long)pg_cache->committed_page_index.nr_committed_pages,
+              0L,
+              0L,
```

Comment:
I am still studying the new code, so probably I will find the proper answer when I finish the review, but considering that now we are having constants as arguments, we probably can move them inside the format instead to force function to assign values for it later like normal [vfprintf](https://android.googlesource.com/platform/bionic/+/9831ad3/libc/stdio/vfprintf.c) does.